helm/prometheus-rules/templates/recording-rules/service-level.rules.yml

```diff
@@ -38,6 +38,45 @@ spec:
         service: api-server
       record: slo_target
 
+      # -- daemonset
+    - expr: |
+        label_replace(
+          kube_daemonset_status_desired_number_scheduled{cluster_id!="argali|giraffe", namespace=~"giantswarm|kube-system", daemonset=~"aws-node|kiam-server|kiam-agent"},
```

Comment:
Do we also want to exclude akita?

Reply:
Right now it is not excluded so I just kept it that way. I personally think that we don't really need that distinction anymore anyway, especially because it does not affect workload clusters. Let's see if we see issues with `akita` that makes us think differently.